layouts/partials/ecosystem/registry/entry.html

```diff
@@ -167,7 +169,17 @@ <h6><i class="fa-solid fa-forward"></i> Quick Install</h6>
                 <div class="fw-bold">
                     {{ $languageNames.Get . | default (humanize .) }}
                 </div>
-                <small>Language</small>
+                {{ if eq (replace (lower .) " " "") "collector" }}
+                    <small data-bs-toggle="tooltip" 
+                        title="Collector: This is an extension for the OpenTelemetry Collector.">
+                        <a href="https://opentelemetry.io/docs/collector/" target="_blank" class="text-muted">Collector</a>
```

Comment:
```suggestion
                        Collector
```

skip the link for now